runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -458,6 +464,28 @@ public void onChatMessage(ChatMessage event)
 		{
 			removeGameTimer(STAFF_OF_THE_DEAD);
 		}
+
+		if (config.showFreezes() && event.getMessage().equals("<col=ef1020>You have been frozen!</col>"))
```

Comment:
After #4072 has been merged, this string now should be constant at the top of the file

Reply:
Personally, I'd just be happy with the string being a constant at the top of the file.